RoosterPlanner.Api/Controllers/PersonsController.cs

```diff
@@ -35,22 +36,24 @@ public PersonsController(IMapper mapper, IPersonService personService, IProjectS
         [HttpGet("{id}")]
         public async Task<ActionResult> Get(Guid id)
         {
-            PersonViewModel personVm = new PersonViewModel();
+            PersonViewModel personVm = null;
 
             try
             {
-                TaskResult<Person> result = null;//await this.personService.Get(id);
+                TaskResult<User> result = await this.personService.GetUser(id);
                 if (result.Succeeded)
                 {
-                    //projectDetailsVm = this.mapper.Map<ProjectDetailsViewModel>(result.Data);
+                    personVm = PersonViewModel.CreateVm(result.Data);
```

Comment:
Gebruikt static methode is goed. Maar als je 'mapper' (Automapper) niet gebruikt, dan deze niet meer injecteren in constructor.